examples/automatic_speech_recognition/automatic_speech_recognition/evaluator.py

```diff
@@ -0,0 +1,301 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+import langcodes
+import langid
+import numpy as np
+from jiwer import cer
+from jiwer import process_words
+from numwords_to_nums.numwords_to_nums import NumWordsToNum
+from workflow import GroundTruth
+from workflow import Inference
+from workflow import TestCase
+from workflow import TestCaseMetric
+from workflow import TestSample
+from workflow import TestSampleMetric
+from workflow import TestSuiteMetric
+
+from kolena.workflow import AxisConfig
+from kolena.workflow import Curve
+from kolena.workflow import CurvePlot
+from kolena.workflow import Histogram
+from kolena.workflow import Plot
+from kolena.workflow.evaluator_function import EvaluationResults
+from kolena.workflow.evaluator_function import TestCases
+from kolena.workflow.annotation import ClassificationLabel
+
+from utils import generate_diff_word_level, preprocess_transcription
+
+
+def compute_test_sample_metrics(gt: GroundTruth, inf: Inference) -> TestSampleMetric:
+    gt = preprocess_transcription(gt)
+    inf = preprocess_transcription(inf)
+
+    word_fp, (fn_count, fp_count, ins_count, del_count, sub_count, sub_list, ins_list, del_list) = generate_diff_word_level(gt, inf, mode="fp")
+    word_fn, _ = generate_diff_word_level(gt, inf, mode="fn")
+
+    wer_metrics = process_words(gt, inf)
+    word_errors = ins_count + sub_count + del_count
+    word_error_rate = wer_metrics.wer
+    match_error_rate = wer_metrics.mer
+    word_information_lost = wer_metrics.wil
+    word_information_preserved = wer_metrics.wip
+    character_error_rate = cer(gt, inf)
+
+    language = langcodes.Language.get(langid.classify(inf)[0]).display_name()
+
+    return TestSampleMetric(
+        WordErrors=word_errors,
+        WordErrorRate=word_error_rate,
+        MatchErrorRate=match_error_rate,
+        WordInformationLost=word_information_lost,
+        WordInformationPreserved=word_information_preserved,
+        CharacterErrorRate=character_error_rate,
+        FalsePositiveText=word_fp,
+        FalseNegativeText=word_fn,
+        FNCount=fn_count,
+        FPCount=fp_count,
+        InsertionCount=ins_count,
+        DeletionCount=del_count,
+        SubstitutionCount=sub_count,
+        Language=language,
+
+        Substitutions = [ClassificationLabel(item) for sublist in sub_list for item in (sublist if isinstance(sublist, list) else [sublist])],
```

Comment:
hmm I think this line definitely exceeds the max char limit.  can you re-run the linter?